tests/qmltests/Stage/tst_PhoneStage.qml

```diff
@@ -583,5 +589,21 @@ Item {
             compare(topLevelSurfaceList.idAt(0), webbrowserSurfaceId);
             compare(webbrowserApp.focused, true);
         }
+
+        /*
+            Check that set & unset allowInteractivity won't change main app.
+        */
+        function test_interactivityShouldNotChangeMainApp()
+        {
+            // Have something to focus
+            var webbrowserSurfaceId = topLevelSurfaceList.nextId;
+            var webbrowserApp  = ApplicationManager.startApplication("morph-browser");
```

Comment:
double space

Reply:
Indeed that's copied & pasted from the code above. Anyway, that's fixed (in my test case).